server.py

```diff
@@ -204,26 +204,33 @@ def delete_product(id):
       return make_response(jsonify(message), rc)
 
 
-@app.route('/inventory/products/', methods= ['POST'])
+@app.route('/inventory/products', methods= ['POST'])
 def create_products():
     """
 
     This method will create a storage for a new product
 
-    Args:
+    Args: data with restock_level provided.
 
     Returns:
 
-      response: create successful message with status 201 if succeeded, the auto assigned product id should also be presented
-                or invalid update with status 400 if the create request violates any limitation
-                or conflict update with status 409 there is a identical product already in the data.
+      response: create successful message with status 201 if succeeded,
+                the auto assigned product id and location id should also be presented
+                or invalid create with status 400 if the create request violates any limitation
 
     Todo:
       * Finish the implementations.
       * Write the tests for this.
 
     """
-    pass
+    data = request.get_json()
+    if RESTOCK_LEVEL in data and type(data[RESTOCK_LEVEL]) is int and data[RESTOCK_LEVEL] > 0:
```

Comment:
Good suggestion!
But since we'll just be using int temporarily(string for id is more reasonable)
Maybe we can keep this and update in future if needed?